server/filters/fallbackMessageFilter.test.ts

```diff
@@ -0,0 +1,28 @@
+import fallbackMessageFilter from './fallbackMessageFilter'
```

Comment:
I wrote a "fallback message filter", whose job is to render some fallback message if the main value is null/blank etc.
You can see how it's used further down in the nunjucks template, but it means we don't need to put if statements or logic in the template 👍 , and we can test the filter once in isolation here 👍